CaesarCipher.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package caesarcipher;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
+ * @author kailabillie
+ */
+public class CaesarCipher {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+
+        String encodedString = Encode("-1:some text");
+        System.out.println(encodedString);
+
+    }
+
+    static String Encode(String word) {
+
```

Comment:
add line 29 to handle null/empty edge cases
`if (Objects.equals(word,null) || word.isEmpty()) 
     throw new IllegalArgumentException("empty/uninitialized string");`